rest/api_test.go

```diff
@@ -3121,6 +3121,8 @@ func TestAddingAttachment(t *testing.T) {
 	rt := NewRestTester(t, nil)
 	defer rt.Close()
 	defer func() { walrus.MaxDocSize = 0 }()
+	// Generate RT config for skip TLS when running with couchbases:// protocol
+	rt.Bucket()
```

Comment:
Why is this now required? This should get called when a bucket is requested by the test? This is implicitly done in this case when createDoc is called. Which triggers an admin request.